staff/diego-lopez/course-1/task2/design_1/styles.css

```diff
@@ -0,0 +1,62 @@
+body{
```

Comment:
hey, format your code

Reply:
Question: I have been reading about code formating, and in CSS it says that the sons must be in alphabetical order but it doesn't say how should the fathers should be ordered, is there a rule for this at all?